tools/rtpsrelay/PublicationListener.cpp

```diff
@@ -6,10 +6,10 @@ namespace RtpsRelay {
 
 PublicationListener::PublicationListener(OpenDDS::DCPS::DomainParticipantImpl* participant,
                                          WriterEntryDataWriter_ptr writer,
-                                         const AssociationTable& association_table) :
+                                         const RelayAddresses& relay_addresses) :
```

Comment:
```suggestion
                                         const RelayAddresses& relay_addresses)
```
(see ReaderListener::ReaderListener)